src/gui/qgsquerybuilder.h

```diff
@@ -59,6 +60,12 @@ class GUI_EXPORT QgsQueryBuilder : public QDialog, private Ui::QgsQueryBuilderBa
     QString sql();
     void setSql( const QString &sqlStatement );
 
+    /**
+     * Returns the code editor widget for the SQL.
+     * \since QGIS 3.18
+     */
+    QgsCodeEditor *codeEditorWidget() const { return txtSQL; }
```

Comment:
Off topic, but can you rename the `txtSQL` to `mTxtSql` to be coding standard compliant?

Reply:
done